modules/tinymce/src/core/test/ts/browser/EditorTest.ts

```diff
@@ -327,113 +325,147 @@ UnitTest.asynctest('browser.tinymce.core.EditorTest', function (success, failure
+    Assert.eq('setMode', 2, clickCount);
+  }));
 
-  suite.test('translate', function (editor) {
+  const sTranslate = (editor: Editor) => Log.step('', 'translate', Step.sync(() => {
     EditorManager.addI18n('en', {
       'input i18n': 'output i18n',
       'value:{0}{1}': 'value translation:{0}{1}'
     });
 
-    LegacyUnit.equal(editor.translate('input i18n'), 'output i18n');
-    LegacyUnit.equal(editor.translate([ 'value:{0}{1}', 'a', 'b' ]), 'value translation:ab');
-  });
+    Assert.eq('translate', 'output i18n', editor.translate('input i18n'));
+    Assert.eq('translate', 'value translation:ab', editor.translate([ 'value:{0}{1}', 'a', 'b' ]));
+  }));
 
-  suite.test('Treat some paragraphs as empty contents', function (editor) {
+  const sTreatSomeParagraphsAsEmptyContents = (editor: Editor) => Log.step('', 'Treat some paragraphs as empty contents', Step.sync(() => {
     editor.setContent('<p><br /></p>');
-    LegacyUnit.equal(editor.getContent(), '');
+    Assert.eq('Treat some paragraphs as empty contents', '', editor.getContent());
 
     editor.setContent('<p>\u00a0</p>');
-    LegacyUnit.equal(editor.getContent(), '');
-  });
+    Assert.eq('Treat some paragraphs as empty contents', '', editor.getContent());
+  }));
 
-  suite.test('kamer word boundaries', function (editor) {
+  const sKamerWordBoundaries = (editor: Editor) => Log.step('', 'kamer word boundaries', Step.sync(() => {
     editor.setContent('<p>!\u200b!\u200b!</p>');
-    LegacyUnit.equal(editor.getContent(), '<p>!\u200b!\u200b!</p>');
-  });
+    Assert.eq('kamer word boundaries', '<p>!\u200b!\u200b!</p>', editor.getContent());
+  }));
 
-  suite.test('Preserve whitespace pre elements', function (editor) {
+  const sPreserveWhitespacePreElements = (editor: Editor) => Log.step('', 'Preserve whitespace pre elements', Step.sync(() => {
     editor.setContent('<pre> </pre>');
-    LegacyUnit.equal(editor.getContent(), '<pre> </pre>');
-  });
+    Assert.eq('kamer word boundaries', '<pre> </pre>', editor.getContent());
+  }));
 
-  suite.test('hasFocus', function (editor) {
+  const sHasFocus = (editor: Editor) => Log.step('', 'hasFocus', Step.sync(() => {
     editor.focus();
-    LegacyUnit.equal(editor.hasFocus(), true);
+    Assert.eq('hasFocus', true, editor.hasFocus());
 
     const input = document.createElement('input');
     document.body.appendChild(input);
 
     input.focus();
-    LegacyUnit.equal(editor.hasFocus(), false);
+    Assert.eq('hasFocus', false, editor.hasFocus());
 
     editor.focus();
-    LegacyUnit.equal(editor.hasFocus(), true);
+    Assert.eq('hasFocus', true, editor.hasFocus());
 
     input.parentNode.removeChild(input);
-  });
-
-  TinyLoader.setup(function (editor, onSuccess, onFailure) {
-    Pipeline.async({}, suite.toSteps(editor), function () {
-      onSuccess();
-    }, onFailure);
+  }));
+
+  const sHasPlugin = (editor: Editor) => Log.step('', 'hasPlugin', Step.sync(() => {
```

Comment:
This should really have some more tests. For example to make sure it doesn't match `list` in `advlist`. It's also not checking the comma case you've included in the code.

- [ ] Add more tests please